share/elections/2024-01.md

```diff
@@ -2,13 +2,11 @@
 
 This page contains the metadata about the 2024 VoxPupuli Elections.
 
-
-# Scope
+## Scope
```

Comment:
We usually have one file per year. Did you update the file from last year intentionally, or do you want to commit it as 2025-01.md?